### ISSUE TITLE
New header not installed

### ISSUE THREAD

**Issue by bska** [Ensembles/ert#851]
Since PR #842 I'm getting build failures of the form
```
/home/bska/work/opm/inst/dbg/include/ert/util/buffer.h:101:27: fatal error: buffer_string.h: No such file or directory
 #include "buffer_string.h"
                           ^
```
in client code that uses the Ert.  I'm **guessing** that `buffer_string.h`, being a new header in 4b215d2, was just overlooked and that it should be installed despite being a deprecated interface.

**Comment by joakim-hove** [comment]
> in client code that uses the Ert. I'm guessing that buffer_string.h,
being a new header in 4b215d2, was just overlooked and that it should be
installed despite being a deprecated interface.

You are right - fix coming.

**Comment by bska** [comment]
Closing as fixed in #853.
